app/Locale/German.php

```diff
@@ -55,6 +55,11 @@ public function get_locale(): array
 			'SET_MAP_PROVIDER' => 'Speichere Provider für OpenStreetMap Karten',
 			'FULL_SETTINGS' => 'Alle Einstellungen',
 			'UPDATE' => 'Update',
+			'RESET_TOKEN' => 'API-Schlüssel zurücksetzen',
+			'RESET' => 'Zurücksetzen',
+			'DISABLE' => 'Deaktivieren',
+			'ENABLE_TOKEN' => 'API-Schlüssel aktivieren',
+			'DISABLED' => 'deaktiviert',
```

Comment:
I just had a closer look here, because the German translation for the key `DISABLED` does not work in the frontend as expected. But this is a bug in the frontend. Nonetheless another thought came to my mind.

Maybe we should use more specific keys here in order to avoid a clash of keys or strange keys in the future. Something more like

```suggestion
			'DISABLE_TOKEN_TOOLTIP' => 'Deaktivieren',
			'ENABLE_TOKEN' => 'API-Schlüssel aktivieren',
			'DISABLED_TOKEN_STATUS_MSG' => 'Deaktiviert',
```